src/views/SystemIntake/Documents/DocumentsTable.tsx

```diff
@@ -0,0 +1,320 @@
+      </PageHeading>
+      <p className="margin-top-1 font-body-md line-height-body-5 tablet:grid-col-12 desktop:grid-col-8">
+        {t('intake:documents.tableDescription')}
+      </p>
+
+      <h4 className="margin-bottom-1 margin-top-5">
+        {t('intake:documents.tableTitle')}
+      </h4>
+
+      <Button
+        className="margin-bottom-1"
+        type="button"
+        onClick={() =>
+          history.push(`/system/${systemIntake.id}/documents/upload`)
+        }
+      >
+        {t(
+          `technicalAssistance:documents.${
+            documents.length ? 'addAnotherDocument' : 'addDocument'
+          }`
+        )}
+      </Button> */}
+
+      <ConfirmDeleteModal />
+
+      <Table bordered={false} fullWidth scrollable {...getTableProps()}>
+        <thead>
+          {headerGroups.map(headerGroup => (
+            <tr {...headerGroup.getHeaderGroupProps()}>
+              {headerGroup.headers.map((column, index) => (
+                <th
+                  {...column.getHeaderProps(column.getSortByToggleProps())}
+                  aria-sort={getColumnSortStatus(column)}
+                  scope="col"
+                  className="border-bottom-2px"
+                >
+                  <Button
+                    type="button"
+                    unstyled
+                    className="width-full display-flex"
+                    {...column.getSortByToggleProps()}
+                  >
+                    <div className="flex-fill text-no-wrap">
+                      {column.render('Header')}
+                    </div>
+                    <div className="position-relative width-205 margin-left-05">
+                      {getHeaderSortIcon(column)}
+                    </div>
+                  </Button>
+                </th>
+              ))}
+            </tr>
+          ))}
+        </thead>
+        <tbody {...getTableBodyProps()}>
+          {rows.map(row => {
+            prepareRow(row);
+            return (
+              <tr {...row.getRowProps()}>
+                {row.cells.map((cell, index) => {
+                  return (
+                    <td {...cell.getCellProps()}>{cell.render('Cell')}</td>
+                  );
+                })}
+              </tr>
+            );
+          })}
+        </tbody>
+      </Table>
+
+      {!documents.length && <p>{t('intake:documents.noDocuments')}</p>}
+
+      {/* <Pager
+        className="margin-top-6"
+        next={{
+          text: t(
+            documents.length > 0
+              ? 'Next'
+              : 'intake:documents.continueWithoutDocuments'
+          ),
+          outline: documents.length === 0,
+          onClick: () => history.push(`/system/${systemIntake.id}/review`)
+        }}
+        back={{
+          onClick: () =>
+            history.push(`/system/${systemIntake.id}/contract-details`)
+        }}
+        taskListUrl={`/governance-task-list/${systemIntake.id}`}
+        border={false}
+      /> */}
+    </div>
+  );
```

Comment:
This might be old/copied code that can be removed.  If it isn't, it might need a comment to explain it's future intended use

Reply:
Thanks for catching this! I commented this out while refactoring and forgot to remove it.